app/views/application/contact-details/address.njk

```diff
@@ -10,9 +10,6 @@
 {% endif %}
 
 {% block pageNavigation %}
-  {{ govukBreadcrumbs({
-    items: applicationBreadcrumbItems
-  }) }}
 {% endblock %}
```

Comment:
Empty `pageNavigation` block?

Reply:
Something needs to go in here eventually. I just removed this so I could safely delete `applicationBreadcrumbItems`

Reply:
```suggestion
  {# TODO: Add back link #}
{% endblock %}
```

Reply:
I've deleted it